test/runtime/services/multimediad/service.test.js

```diff
@@ -0,0 +1,50 @@
+var test = require('tape')
+var Service = require('/usr/yoda/services/multimediad/service')
+
+function Light () {
+
+}
+Light.prototype.invoke = function () {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve()
+    }, 500)
+  })
+}
+
+function getServiceInstance (Light) {
```

Comment:
If you were to use different light mocker here, it would be better that use another name than `Light`.